spec/counter_culture_spec.rb

```diff
@@ -945,6 +946,14 @@
     product.twitter_reviews_count.should == 1
   end
 
+  it "handles an inherited STI counter cache correctly", focus: true do
```

Comment:
If you remove the `focus: true` here you could keep the config in `spec_helper` for your convenience. That way you can focus any spec you're working on.